### DISCUSSION TITLE
Using new Image() in next.js

### DISCUSSION THREAD

**Discussion by danielpaz6** [vercel/next.js#13195] [Help] [answered]
Hi,

I'm working with canvas and in someplace in my code I needed to make an instance of `Image`:

`const e = new Image;`

but it says that Image not exists, after some searching on the internet the reason might be that it tries to run in on the server-side?

**Comment by jamesmosier** [comment, marked as answer]
`Image` is browser only, so you'll want to wait to run this code behind a check for `if (typeof window !== 'undefined') { /* create your image */ }`

Otherwise you could dynamically load the module that performs this logic [as seen here](https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr)

**Comment by webdiego** [comment]
I have the same problem, is not working with `if (typeof window !== 'undefined') { /* create your image */ }` and even inside an useEffect
